src/components/UniversalTable/UniversalTable.js

```diff
@@ -0,0 +1,55 @@
+import "./UniversalTable.css";
+
+export default function UniversalTable(collection, options) {
+  this.rowClick = options.onClick;
+  this.collection = collection;
+  this.parent = options.parent;
+  this.headerCells = options.header;
+}
+
+UniversalTable.prototype.render = function () {
+  const tableHeader = document.createElement("ul");
+
+  const tableHeaderRow = document.createElement("li");
+
+  tableHeaderRow.innerHTML = this.headerCells.reduce((acc, currentValue) => {
```

Comment:
consider [this](https://github.com/BUI-studies/kurkulyator/pull/17/files#r1359833062) and rewrite the render of the header row